src/main/java/Duke.java

```diff
@@ -1,10 +1,190 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
```

Comment:
Cannot write everything in main function. Even if it's not main function, this function itself is too long also, which reduce the readability of the code.